docs/packages.md

```diff
@@ -67,115 +67,197 @@ during the build.
+
+#### Examples
+
+##### Simple build
+
+There are no contexts starting with `linuxkit-`, no environment variable `LINUXKIT_BUILDERS`, no command-line argument `--builders`.
+
+linuxkit will build any requested packages using `docker buildx` on the local platform, with a builder (created, if necessary) named `linuxkit`.
+Builds for the same architecture will be native, builds for other platforms will use either qemu or cross-building.
 
-1. Determine the name and tag for the image as follows:
-   * The tag is from the hash of the git tree for that package. You can see it by doing `linuxkit pkg show-tag «path-to-package»`.
-   * The name for the image is from `«path-to-package»/build.yml`
-   * The organization for the package is given on the command-line, default to `linuxkit`.
-1. Build the package in the given path using your local docker instance for the local platform. E.g. if you are running on `linux/arm64`, it will build for `linux/arm64`.
-1. Tag the build image as `«image-name»:«hash»-«arch»`
-1. Push the image to the hub
 
-#### Manifest Only
+##### Specified target
 
-To perform just the manifest steps, do:
+You create a context named `my-remote-arm64` and then run:
 
+```bash
+linuxkit pkg build --platforms=linux/arm64,linux/amd64 --builders=linux/arm64=my-remote-arm64
 ```
-linuxkit pkg push --image=false --manifest «path-to-package»
+
+linuxkit will build:
+
+* for arm64 using the context `my-remote-arm64`, since you specified in `--builders` to use `my-remote-arm64` for `linux/arm64`
+* for amd64 using the context `default` and the `linuxkit` builder, as that is the default fallback
+
+The same would happen if you used `LINUXKIT_BUILDERS=linux/arm64=my-remote-arm64` instead of the `--builders` flag.
+
+##### Named context
+
+You create a context named `linuxkit-arm64` and then run:
+
+```bash
+linuxkit pkg build --platforms=linux/arm64,linux/amd64
 ```
 
-The options do the following:
+linuxkit will build:
 
-* `--image=false` do not push the image, as you already did it; you can, of course, skip this argument and push the image as well
-* `--manifest` create and push the manifest
+* for arm64 using the context `linuxkit-arm64`, since there is a context with the name `linuxkit-<arch>`, and you did not override it using `--builders` or the environment variable `LINUXKIT_BUILDERS`
+* for amd64 using the context `default` and the `linuxkit` builder, as that is the default fallback
 
-This will do the following:
+##### Combination
 
-1. Find all of the images on the hub of the format `«image-name»:«hash»-«arch»`
-1. Create a multi-arch manifest called `«image-name»:«hash»` (note no `-«arch»`)
-1. Push the manifest to the hub
-1. Sign the manifest with your key
+You create a context named `linuxkit-arm64`, and another named `my-remote-builder-amd64` and then run:
+
+```bash
+linuxkit pkg build --platforms=linux/arm64,linux/amd64 --builders=linux/amd64=my-remote-builder-amd64
```

Comment:
nit: perhaps `--builders linux/amd64=my-remote-builder-amd64` - removed `=`.
while valid `--flag=foo=bar` irritates me, but this may be a me thing

Reply:
It should work with or without `=`, so I removed it from docs to make it clearer.